web/modules/custom/ys_base/js/ys_base.js

```
@@ -0,0 +1,76 @@
+
+            // Save to localStorage.
+            localStorage.setItem(storageKey, currentValue.toString());
+
+            // Add visual feedback.
+            valueElement.classList.add('updated');
+            setTimeout(function () {
+              valueElement.classList.remove('updated');
+            }, 300);
+
+            // Log action.
+            console.log('Counter ' + action + ': ' + currentValue);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)